examples/resources/gcore_dns_zone_record/TESTING.md

```diff
@@ -0,0 +1,89 @@
+
+# How test development version
+
+other than golang test that might work but not really reflect the real world, 
+for example import state not caught by golang test.
+
+0. make sure local provider already set, like this:
+```shell
+# or replace whole .terraformrc if necessary 
+tee -a ~/.terraformrc << EOF
+provider_installation {
+  dev_overrides {
+			"local.gcore.com/repo/gcore" = "$(go env GOPATH)/bin"
+			"registry.terraform.io/g-core/gcore" = "$(go env GOPATH)/bin"
+	}
+  direct {}
+}
+EOF
+```
+1. make sure you already compile and set your development env, like this:
+```shell
+make build && mv -v bin/* $(go env GOPATH)/bin/
+```
+2. visit https://accounts.gcore.com/profile/api-tokens
+3create API token and put it on upper part of the `resource.tf`
```

Comment:
format is broken? You can use hash sign `#` in MD files

Reply:
removed